.travis.yml

```diff
@@ -1,5 +1,6 @@
 sudo: false
 language: java
+dist: trusty
```

Comment:
would probably prefer we only use try on jdk 8 instead of wholesale. (move to between line 8-9)

Reply:
done